app/controllers/brexit_checker_controller.rb

```diff
@@ -91,21 +115,17 @@ def enable_caching_unless_accounts
   end
 
   def results_from_account
-    @results_from_account ||=
```

Comment:
I see, so we've moved a lot to helpers or other instance variables [like this](https://github.com/alphagov/finder-frontend/pull/2234/commits/f7559a601390411ee591f8a014e2042374f57c08#diff-1629a008ab38ebefe335fd1ca5dc0b1aR38)

Reply:
Yes, I decided that they should call this direct since it can log the user out.  So the pattern I've gone for now is to call this, then check if the user is still logged in, and then do stuff with the result.